keepalived/vrrp/vrrp_snmp.c

```diff
@@ -107,7 +107,6 @@
 #ifdef _WITH_SNMP_KEEPALIVED_
 /* VRRP SNMP defines */
 enum snmp_vrrp_magic {
-	VRRP_OID KEEPALIVED_OID = 2,
```

Comment:
This is clearly nonsense, and was a bad edit by me in commit c4510e60.

My patch reinstates the definition of VRRP_OID, keeping it in this file.